idb/postgres/postgres_migrations.go

```diff
@@ -46,6 +47,8 @@ func init() {
 		{m7StaleClosedAccounts, false, "clear some stale data from closed accounts"},
 		{m8TxnJSONEncoding, false, "some txn JSON encodings need app keys base64 encoded"},
 		{m9SpecialAccountCleanup, false, "The initial m6 implementation would miss special accounts."},
+
```

Comment:
is there an m10 hiding in another PR?

Reply:
yeah exactly,  I'm hoping to make the merge easier